app/assets/javascripts/application.js

```diff
@@ -255,7 +255,7 @@ function template_info(div, url) {
   // Use a post to avoid request URI too large issues with big forms
   $.ajax({
     type: "POST",
-    url: url + "&provisioning=" + build,
+    url: url + ((url.indexOf('?') == -1) ? '?' : '&') + "provisioning=" + build,
```

Comment:
again - nitpick, however, i believe this is more "correct" e.g. your code will return true if url has a question mark in the middle of the string.